### DISCUSSION TITLE
Need step by step guidance to use tailwind standalone CLI in windows

### DISCUSSION THREAD

**Discussion by kharabela** [tailwindlabs/tailwindcss#13650] [Help] [answered]
_My goal is not to avoid use of node npm
-https://tailwindcss.com/blog/standalone-cli
-Created a folder named as tailwind
-Trying to replicate as shown on website
# Create a tailwind.config.js file
./tailwindcss init

# Start a watcher
./tailwindcss -i input.css -o output.css --watch

# Compile and minify your CSS for production
./tailwindcss -i input.css -o output.css --minify

In windows cmd terminal I get errors
```
C:\Tailwind>./tailwindcss init
'.' is not recognized as an internal or external command,
operable program or batch file.
```

**Comment by wongjn** [comment]
`./` is for unix-based systems, try without this prefix:

```cmd
tailwindcss init
```

Also ensure that the executable name matches, so ensure there is a `tailwindcss.exe` in the directory you call it from. For example, if the executable was named `tailwindcss-windows-x64.exe`, you'd do:

```cmd
tailwindcss-windows-x64 init
```

**Reply by kharabela** [reply]
@wongjn - This is how it works in windows

**Reply by wongjn** [reply, marked as answer]
Great! Looks like you got it working then 👍

**Comment by kharabela** [comment]
This works in Windows Powershell

Download - https://github.com/tailwindlabs/tailwindcss/releases/tag/v3.4.3

Create a folder
static\css

1.input.css
@tailwind base;
@tailwind components;
@tailwind utilities;

2.output.css

# Create a tailwind.config.js file
.\tailwindcss-windows-x64 init
Created Tailwind CSS config file: tailwind.config.js

# Start a watcher
.\tailwindcss-windows-x64 -i static\css\input.css -o static\css\output.css --watch
Browserslist: caniuse-lite is outdated. Please run:
  npx update-browserslist-db@latest
  Why you should do it regularly: https://github.com/browserslist/update-db#readme

Rebuilding...

warn - No utility classes were detected in your source files. If this is unexpected, double-check the `content` option in your Tailwind CSS configuration.
warn - https://tailwindcss.com/docs/content-configuration

Done in 133ms.

# Compile and minify your CSS for production
.\tailwindcss-windows-x64 -i static\css\input.css -o static\css\output.css --minify
Browserslist: caniuse-lite is outdated. Please run:
  npx update-browserslist-db@latest
  Why you should do it regularly: https://github.com/browserslist/update-db#readme

Rebuilding...

warn - No utility classes were detected in your source files. If this is unexpected, double-check the `content` option in your Tailwind CSS configuration.
warn - https://tailwindcss.com/docs/content-configuration

Done in 162ms.
